package.json

```diff
@@ -3,20 +3,20 @@
   "version": "1.0.0",
   "description": "Simorgh",
   "scripts": {
-    "amp:validate": "wait-on http://localhost:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost:7080/igbo.amp",
+    "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp",
     "audit:ci": "audit-ci --low",
-    "bbcA11y": "wait-on -t 20000 http://localhost:7080/news/articles/c6v11qzyv8po && bbc-a11y",
+    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
     "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
     "build": "rm -rf build && cp envConfig/local.env .env && npm run webpack:prod && npm run test:size",
     "build:storybook": "build-storybook -c .storybook -o storybook_dist",
     "build:test": "cp envConfig/test.env .env && npm run webpack:prod",
     "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
-    "cypress": "cypress run",
-    "cypress:interactive": "cypress open",
-    "cypress:3rdParty": "cypress run --project ./3rdPartyCypress/.",
-    "cypress:3rdParty:interactive": "cypress open --project ./3rdPartyCypress/.",
-    "cypress:storybook": "wait-on -t 120000 http://localhost:9001 && cypress run --project ./.storybook/.",
-    "cypress:storybook:interactive": "cypress open --project ./.storybook/.",
+    "cypress": "export NO_PROXY=localhost,localhost.bbc.com && cypress run",
+    "cypress:interactive": "export NO_PROXY=localhost,localhost.bbc.com && cypress open",
+    "cypress:3rdParty": "export NO_PROXY=localhost,localhost.bbc.com && cypress run --project ./3rdPartyCypress/.",
+    "cypress:3rdParty:interactive": "export NO_PROXY=localhost,localhost.bbc.com && cypress open --project ./3rdPartyCypress/.",
+    "cypress:storybook": "export NO_PROXY=localhost,localhost.bbc.com && wait-on -t 120000 http://localhost.bbc.com:9001 && cypress run --project ./.storybook/.",
+    "cypress:storybook:interactive": "export NO_PROXY=localhost,localhost.bbc.com && cypress open --project ./.storybook/.",
```

Comment:
To DRY this up a little could we have one script (e.g. `cypress`) that sets the env and runs the `cypress` binary and then the others all call out to it? e.g. `npm run cypress -- open`

Reply:
👍